novawallet/Common/Helpers/AssetBalance/AssetBalanceFormatterFactory.swift

```diff
@@ -12,10 +12,14 @@ protocol AssetBalanceFormatterFactoryProtocol {
 
     func createTokenFormatter(
         for info: AssetBalanceDisplayInfo,
-        roundingMode: NumberFormatter.RoundingMode
+        roundingMode: NumberFormatter.RoundingMode,
+        useSuffixForBigNumbers: Bool
     ) -> LocalizableResource<TokenFormatter>
 
-    func createAssetPriceFormatter(for info: AssetBalanceDisplayInfo) -> LocalizableResource<TokenFormatter>
+    func createAssetPriceFormatter(
+        for info: AssetBalanceDisplayInfo,
+        useSuffixForBigNumbers: Bool
+    ) -> LocalizableResource<TokenFormatter>
 
     func createTotalPriceFormatter(for info: AssetBalanceDisplayInfo) -> LocalizableResource<TokenFormatter>
```

Comment:
As we have `useSuffixForBigNumber` for `createAssetPriceFormatter` i feel that we don't need `createTotalPriceFormatter` anymore